megamek/src/megamek/server/totalwarfare/MovePathHandler.java

```diff
@@ -125,6 +125,12 @@ class MovePathHandler extends AbstractTWRuleHandler {
     }
 
     void processMovement() {
+        if (md.getMpUsed() > 0) {
+            // All auto-hit hexes for this unit (not including preset targets) are cleared
+            // if any MP are expended.
+            entity.aTracker.clearHitHexMods();
```

Comment:
I think a report would be reasonable for this. But not vital.

Reply:
Sorry, I had stepped away for dinner.
The only issue with reporting is that this is happening during the movement phase, as each move is executed, so reports inserted here would be interleaved with the normal movement reporting.
It feels a little wrong to insert reports about lost auto-hit artillery mods into the move report, but we could store the removed Coords _in_ the ArtilleryTracker and report them during the next Indirect Phase report...
Actually, it might be good to report any auto-hit mods on a per-unit basis as well since we don't do that either.
I'll file an RFE, but I think it's outside the sope of this fix.